### ISSUE TITLE
Bug: Pinned libraries are displayed incorrectly

### ISSUE THREAD

**Issue by AbyssalSoda** [files-community/Files#13308]
### Description

When using Windows library feature, pinned libraries have their icons disappear. 

### Steps To Reproduce

Icon Issue:
1. Create a new library & give it a custom icon
2. Pin it to quick access in File Explorer or Favorites in Files App
3. When in the Files App you will notice the icon is missing and the folder will read as (Name of Library).library-ms
![sacsda](https://github.com/files-community/Files/assets/122791105/283099d5-f89b-4ae5-a2a1-689fd925543b)

### Requirements

Expected Behavior: Favorites bar displays Libraries with their stated name (in my case _Archives_ not _Archives.library-ms_) as well as displaying the selected icon and not an empty folder.

Personal Note: I am still using my old library setup in the Files app. The tag functionality of Files though more versatile than regular libraries lacks the ability for nested tags. Although windows does not have a clean recursive library system, you can essentially create one using library shortcuts; hence most of my files are organized in an intricate nested manner.

### Files Version

2.5.21.0

### Windows Version

10.0.19045.3324

### Log File

[debug.log](https://github.com/files-community/Files/files/12496757/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, I've added this to the project board.

**Comment by yaira2** [comment]
@AbyssalSoda is this an issue if you right click the sidebar and enable the "Library" section?

**Comment by AbyssalSoda** [comment]
> @AbyssalSoda is this an issue if you right click the sidebar and enable the "Library" section?

No, libraries within the libraries folder display the icon and do not have a naming issue, but when pinned to quick favorites the issue arises, the other issues however that I marked as "drag and drop" is present throughout the system regardless.

**Comment by yaira2** [comment]
I can reproduce this issue

**Comment by amphyvi** [comment]
I'm a Libraries user, and the issue is present in the stable Files release for me as well

**Comment by mdtauk** [comment]
Shouldn't Libraries be its own section on the sidebar?  If the user has them turned on in File Explorer on their system?

**Comment by Josh65-2201** [comment]
There's an option for that, but not everyone want all of them shown I guess.

**Comment by mdtauk** [comment]
> There's an option for that, but not everyone want all of them shown/separate I guess.

Eitherway, how we display the library sidebar items, should be used when pinning a library to the pinned section

**Comment by Josh65-2201** [comment]
@AbyssalSoda Can you see if this is still an issue in the current preview version. https://files.community/download
This is working as requested for me

**Comment by yaira2** [comment]
I haven't tested again but I don't think this has been resolved.

**Comment by Josh65-2201** [comment]
The only issue I can see is opening properties from the pin will get the default folders information (Only gets the location correct) rather then the libraries
![Image](https://github.com/user-attachments/assets/e43c4e5c-770d-4cba-a0b9-0d3940960097)

**Comment by yaira2** [comment]
Looks good then
